public/views/new-assignment.html

```diff
@@ -60,6 +60,7 @@
                         <option value="culture">Culture</option>
                         <option value="technology">Technology</option>
                         <option value="politics">Politics</option>
+                        <option value="politics">Sport</option>
                         <option value="entertainment">Entertainment</option>
                         <option value="religion & ideas">Religion & Ideas</option>
```

Comment:
Haven't caused any issues so far, but perhaps we could change to a regular "and". To be fair we haven't tested adding an assignment in that category yet, so you might have a point. Let's change it to "and" in the next PR.